x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -462,7 +380,7 @@ export function copyAllowlistedFields(
 ): TelemetryEvent {
   return Object.entries(allowlist).reduce<TelemetryEvent>((newEvent, [allowKey, allowValue]) => {
     const eventValue = event[allowKey];
-    if (eventValue) {
+    if (null != eventValue) {
```

Comment:
Great catch! What an unusual bug

Reply:
a small thing, but it seems strange to be to see it written this way - most would expect `eventValue != null`.

Also, `eventValue != null` would do what you expect it to, however we generally tend to be more explicit with these types of checks with something like `if (eventValue !== null && eventValue !== undefined)`

Reply:
Yeah, I was wondering about this. `if (eventValue !== null && eventValue !== undefined)` does seem safer. 

```
> null === undefined
false
> null == undefined
true
```

Reply:
> a small thing, but it seems strange to be to see it written this way - most would expect eventValue != null.

Ah.  On the Endpoint team we use [Yoda Conditions](https://en.wikipedia.org/wiki/Yoda_conditions) to avoid a specific type of bug.  I'm happy to reverse it.  "When in Rome..."

Reply:
Regarding `!==`, `eslint` insisted once I added `undefined`.